CodeEdit/Features/CEWorkspace/Models/CEWorkspaceFile+Recursion.swift

```diff
@@ -131,4 +63,31 @@ extension CEWorkspaceFile {
         return topmostParent
     }
 
+#if DEBUG
```

Comment:
Isn't it better to place this if in the function? So that the function is always available if it is called?

Reply:
These were only being used for debugging, so I felt like it was better for it to be compiled conditionally.